src/processors/other/Tremolo.cpp

```diff
@@ -118,29 +121,51 @@ void Tremolo::processAudio (AudioBuffer<float>& buffer)
 {
     const auto numSamples = buffer.getNumSamples();
     waveBuffer.setSize (1, numSamples, false, false, true);
+    modBuffer.setSize (1, numSamples, false, false, true);
```

Comment:
So calling `setSize()` here with the arguments set to avoid reallocating is the right thing to do, but then when does the necessary memory get allocated? There should be another `setSize()` call in the `prepare()` method that allocates the max amount of memory this buffer will need (similar to waveBuffer).

Reply:
Ah right, thanks for catching that and the explanation.